assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -159,10 +173,11 @@ function FontControls({ selectedElements, pushUpdate }) {
         <BoxedNumeric
           aria-label={__('Font size', 'web-stories')}
           value={fontSize}
-          max={PAGE_HEIGHT}
           flexBasis={58}
           textCenter
           onChange={(value) => pushUpdate({ fontSize: value })}
+          min={MIN_MAX.FONT_SIZE.MIN}
```

Comment:
Currently, with ArrowDown the input still goes to minus temporarily (set to 0 on blur though).

Reply:
I think we can fix it in follow up, we need to use min/max values in keyboard interaction (currently you can set negative values in all fields).

Reply:
It should be a very easy addition though in `numeric.js`, to use the passed `min` value to instead of just decreasing it. E.g. something like `newValue = typeof min !== 'undefined' ? Math.max(min, Big(value).minus(diff)) : Big(value).minus(diff);` Thoughts?

However, perhaps I'm missing something and it's not that trivial, we can skip it then and follow up.

Reply:
Added that for a good start.